space-plugins/nuxt-starter/layouts/default.vue

```diff
@@ -1,14 +1,10 @@
 <script setup lang="ts">
 const config = useAppConfig();
-const { completed, appBridgeAuth, oauth } = useAppBridge();
+const appBridge = useAppBridge();
 const nuxtApp = useNuxtApp();
-nuxtApp.provide('appBridge', {
-	completed,
-	appBridgeAuth,
-	oauth,
-});
+nuxtApp.provide('appBridge', appBridge);
 </script>
 
 <template>
-	<slot v-if="!config.appBridge.enabled || completed" />
+	<slot v-if="!config.appBridge.enabled || appBridge.completed.value" />
```

Comment:
As there's a possibility to add more to the return of useAppBridge, I provide the whole appBridge instead of spreading and passing everything.

Because of that change, I had to put `appBridge.completed.value` in the v-if.

(Not `appBridge.completed`, because `appBridge.completed` is still a ref object there. Let me know if you don't understand why it's still not a ref object there)